server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/burstcoin/wallets/burstwallet/BurstWallet.java

```diff
@@ -111,15 +107,15 @@ public String sendCoins(String destinationAddress, BigDecimal amount, String cry
 
         BigInteger recipientInt = BurstAddressValidator.getAccountIdFromRS(destinationAddress);
         if (recipientInt == null) {
-            log.debug("Invalid destination address");
+            log.debug("Invalid destination address"); // Error?
```

Comment:
Should this be `log.error()`?

Reply:
It could be switched, I was using the existing examples...

Reply:
I think error is more appropriate